main.py

```diff
@@ -27,14 +28,52 @@
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(50))
-    email = db.Column(db.String(50), unique=True)
-    password = db.Column(db.String(80))
+    name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(50), unique=True, nullable=False)
+    password = db.Column(db.String(80), nullable=False)
+    events = db.relationship('Event', backref='user', lazy=True)
+    cameras = db.relationship('Camera', backref='user', lazy=True)
```

Comment:
These establish the foreign key relationships in the `event` and `camera` tables. This allows us to do something like `some_event.user` to get to that specific user related to that event.

More info here: https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/